### PR TITLE
Remove embedded and add fluentd target

### PR DESCRIPTION
**Pull request by n313893254**
https://github.com/rancher/rancher/issues/14652
https://github.com/rancher/rancher/issues/14651

### CODE REVIEW

**Comment by n313893254** [comment]
Backend PR: https://github.com/rancher/rancher/pull/14746

File: lib/logging/addon/components/logging/new-edit/component.js [hunk 1]
```diff
@@ -6,22 +6,23 @@ import NewOrEdit from 'ui/mixins/new-or-edit';
...
+  globalStore:      service(),
+  intl:             service(),
   // input
   errors:            null,
   targetType:        null,
   configMap:         null,
-  pageScope:   reads('scope.currentPageScope'),
-  cluster:     reads('scope.currentCluster'),
-  project:     reads('scope.currentProject'),
+  endpointValidate: true,
```
Comment (westlywright) [inline lib/logging/addon/components/logging/new-edit/component.js]: Alignment should be fixed

File: lib/logging/addon/components/logging/new-edit/component.js [hunk 2]
```diff
@@ -34,11 +35,11 @@ export default Component.extend(NewOrEdit, {
...
-        elasticsearchConfig: null,
-        splunkConfig:        null,
-        embeddedConfig:      null,
-        kafkaConfig:         null,
-        syslogConfig:        null,
+        elasticsearchConfig:    null,
+        splunkConfig:           null,
+        kafkaConfig:            null,
+        syslogConfig:           null,
+        fluentForwarderConfig: null,
```
Comment (westlywright) [inline lib/logging/addon/components/logging/new-edit/component.js]: Alignment should be fixed

File: lib/logging/addon/components/logging/target-flentd/component.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        "pod_name": "cattle-6b4ccb5b9d-v57vw",
+        "pod_id": "30c685d0-fa43-11e7-b992-00163e016dc2",
+        "labels": {
+            "app": "cattle",
+            "pod-template-hash": "2607761658"
+        },
+        "host": "47.52.113.251",
+        "master_url": "https://10.233.0.1:443/api"
+    },
+${ fieldsStr }
```
Comment (westlywright) [inline lib/logging/addon/components/logging/target-flentd/component.js]: Alignment should be fixed

File: lib/logging/addon/components/logging/target-flentd/component.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  ...
+}`;
+
+    return template
+  }.property('fieldsStr'),
```
Comment (westlywright) [inline lib/logging/addon/components/logging/target-flentd/component.js]: use `computed` rather then property extensions

File: lib/logging/addon/components/logging/target-flentd/component.js [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+  fieldsStr: function() {
+    const keyValueMap = get(this, 'model.outputTags')
+
+    if (!keyValueMap) {
+      return '';
+    }
+
+    return Object.keys(keyValueMap).map((key) => `    "${ key }": "${ keyValueMap[key] }"`).join(',\n');
+  }.property('model.outputTags'),
```
Comment (westlywright) [inline lib/logging/addon/components/logging/target-flentd/component.js]: use `computed` rather then property extensions

File: lib/logging/addon/components/logging/target-flentd/template.hbs [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+          <div class="row">
+            <div class="col span-12">
+              <label class="acc-label" for="">{{t 'loggingPage.fluentd.weight.label'}}
+              </label>
+              {{input-integer class="form-control input-sm public"
+                              min="0"
+                              value=item.weight
+                              placeholder=(t 'loggingPage.fluentd.weight.placeholder')
+              }}
+              {{!-- <p class="text-info text-small">{{t 'loggingPage.fluentd.weight.helpText'}}</p> --}}
```
Comment (westlywright) [inline lib/logging/addon/components/logging/target-flentd/template.hbs]: Remove comments if no longer needed.

File: lib/logging/addon/components/logging/target-flentd/template.hbs [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+            </div>
+          </div>
+          <div class="row">
+            <div class="col span-12 pt-0" style="margin-top: 43px;">
+              {{input type="checkbox" classNames="form-control" checked=item.standby}}
+              {{t 'loggingPage.fluentd.standby.label'}}
+              {{!-- <p class="text-info text-small" style="margin-left: 25px;">{{t 'loggingPage.fluentd.standby.helpText'}}</p> --}}
```
Comment (westlywright) [inline lib/logging/addon/components/logging/target-flentd/template.hbs]: Remove comments if no longer needed.

File: lib/logging/addon/components/logging/targets-nav/component.js [hunk 8]
```diff
@@ -62,9 +64,13 @@ export default Component.extend({
...
-    ].filter((item) => {
-      return this.get('isClusterLevel') || item.type !== 'embedded';
-    });
+      {
+        type:      'fluentForwarder',
+        css:       `fluentd${   this.currentCss('fluentd') }`,
+        available: true,
+        disabled:  false,
+      },
+    ];
```
Comment (westlywright) [inline lib/logging/addon/components/logging/targets-nav/component.js]: use `computed` rather then property extensions

File: lib/logging/addon/mixins/logging-model.js [hunk 9]
```diff
@@ -64,7 +67,10 @@ export default Mixin.create({
     if (kafka) {
       return 'kafka';
     }
+    if (fluentd) {
+      return 'fluentForwarder'
+    }
 
     return DEFAULT_TARGET_TYPE;
-  }.property('embeddedConfig', 'elasticsearchConfig', 'splunkConfig', 'kafkaConfig', 'syslogConfig'),
+  }.property('elasticsearchConfig', 'splunkConfig', 'kafkaConfig', 'syslogConfig', 'fluentForwarderConfig'),
```
Comment (westlywright) [inline lib/logging/addon/mixins/logging-model.js]: use `computed` rather then property extensions

**Comment by n313893254** [comment]
@westlywright  Done. Thanks for your review.

**Comment by n313893254** [comment]
Rebased, you can only review https://github.com/rancher/ui/pull/2177/commits/ed240cfb46ea4fdf924d0ebf6dcfaac7f2c31aa0

**Comment by aiwantaozi** [comment]
@westlywright Hi West, backend have merged https://github.com/rancher/rancher/pull/14746, could help with this PR, thank you.

**Pull request merged**